docs/user_guide/HOWTO/Upgrade_SIMP.rst

```diff
@@ -99,3 +99,23 @@ of your new Puppet server into the existing environment.
 
    You should *always* start migration with a small number of
    **least critical** clients!
+
+Significant Updates
+-------------------
+
+Upgrading From SIMP-6.0.0-0 to SIMP-6.0.0++
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Puppetserver auth.conf
+""""""""""""""""""""""
+
+Legacy auth.conf has been deprecated.  The puppetserver's auth.conf is now
+managed by Puppet.
+
+1. Back up legacy puppet auth.conf ``<puppet confdir>/auth.conf``, before
```

Comment:
Not sure what you're trying to say here. Auth.conf moved. Do you want me to fullpath the confdir?